chat-commands.js

```diff
@@ -623,13 +623,18 @@ const commands = {
 	},
 	makechatroomhelp: [`/makechatroom [roomname] - Creates a new room named [roomname]. Requires: & ~`],
 
+	subroomgroupchat: 'makegroupchat',
 	makegroupchat: function (target, room, user, connection, cmd) {
 		if (!user.autoconfirmed) {
 			return this.errorReply("You must be autoconfirmed to make a groupchat.");
 		}
 		if (!user.trusted) {
 			return this.errorReply("You must be global voice or roomdriver+ in some public room to make a groupchat.");
 		}
+		if (cmd === 'subroomgroupchat' && (!user.can('mute', null, room) || room.isPrivate)) {
+			return this.errorReply("You can only create subroom groupchats for public rooms you're staff in.");
```

Comment:
Why "public rooms" specifically?

Reply:
Do you want me to allow it for private rooms as well? Seemed pointless but why not?

Reply:
My perspective is definitely "why not".

We can restrict it to public rooms if it becomes a problem.